backend/controllers/retentionReasons.js

```diff
@@ -11,18 +10,16 @@ const retentionReasonsFactory = (elite2Api, dataComplianceApi, logError) => {
 
   const getOffenderUrl = offenderNo => `${dpsUrl}offenders/${offenderNo}`
 
-  const formatDate = date => date && formatTimestampToDate(date)
-
-  const formatDateTime = dateTime => dateTime && formatTimestampToDateTime(dateTime)
-
   const getFormAction = offenderNo => `/offenders/${offenderNo}/retention-reasons`
 
   const getLastUpdate = existingRecord =>
-    existingRecord && {
-      version: existingRecord.etag,
-      timestamp: formatDateTime(existingRecord.modifiedDateTime),
-      user: existingRecord.userId,
-    }
+    existingRecord
+      ? {
+          version: existingRecord.etag,
+          timestamp: formatTimestampToDateTime(existingRecord.modifiedDateTime),
+          user: existingRecord.userId,
+        }
+      : {}
 
   const flagReasonsAlreadySelected = (retentionReasons, existingRecord) => {
     const existingReasons = existingRecord && existingRecord.retentionReasons
```

Comment:
Rather than checking twice for existence here and in the `matchedReason` bit below - this could always return an empty list